Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -654,6 +654,21 @@
 # or (if available) from eos. the number of events limits 
 # the files used as input
 
+###2025 
+pds_2025  = ['BTagMu', 'DisplacedJet', 'EGamma0', 'HcalNZS', 'JetMET0', 'Muon0', 'MuonEG', 'NoBPTX', 'ParkingDoubleMuonLowMass0', 'ParkingHH', 'ParkingLLP', 'ParkingSingleMuon0', 'ParkingVBF0', 'Tau', 'ZeroBias','JetMET1']
```

Comment:
is it possible to add the `ScoutingPFMonitor` PD here and do in a more rational way https://github.com/cms-sw/cmssw/pull/47327 (thus solving https://github.com/cms-sw/cmssw/issues/47315)?

Reply:
Sure, do I remember correctly and for 2024 this would not be possible since some inputs are missing? To understand if I have to touch 2024 too.

Reply:
It's included now, for testing it here we'll have to wait the RAW are copied to disk to CERN.